Changed the navbar to 60px and changed other properties (listed in the description) to suit the new navbar height

ChrisSJung opened the pull request:
Changed the --header-height in global.css from 38px to 60px so the navbar's height reflects as 60px.

Changed the margin-top of the searchBar so it fits the new navbar height.

Changed the nav-logo positioning that was mentioned by bbsmooth in the comment. Now, when max-width:980px is triggered it does not have any awkward state where FCC logo is out of position and the FCC logo and search bar spacing is proper.  

Checklist:
- [x ] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [ x] My pull request targets the `main` branch of freeCodeCamp.
- [ x] I have tested these changes either locally on my machine, or GitPod.

ChrisSJung committed:
Changed the --header-height in global.css to 60px

Changed the --header-height in global.css under layouts from 38px to 60px

ChrisSJung committed:
Changed the positioning of the searchBar in searchbar.css

Changed the position of the searchBar in searchbar.css to make the searchBar be aligned in the center with the new height of navbar that was changed from 38px to 60px.

bbsmooth commented:
I think there is a little more to do here: 

- The language menu button is now taller than the others
- As you narrow the viewport the FCC logo is not quite positioned correctly.

ChrisSJung committed:
Update the navbar to properly fit the width change of the page

The navigation bar now properly adjusts based on the max-width and min-width. Prior to the change, there was an awkward state in the middle of the min-width and max-width. 

Changed the nav-logo to be in the proper position when the max-width:980px is triggered. Now, the navbar is responsive to all size.

ChrisSJung commented:
@bbsmooth Hello, Thank you for the feedback. I made the edits while you were commenting. Now, the FCC logo is in the right place through any viewport. I will change the button height soon.

ChrisSJung committed:
Updated universal-nav.css so button size align FCC Logo stay in right place

Changed few lines so button size align and FCC logo stay in right places throughout different viewports

ChrisSJung commented:
The coding is all done except for the fail in the lint:prettier. Would love feedback. 

Everything else is working.

1. Navbar height changed to 60px;
2. FCC Logo remains in right places throughout different viewports
3. Searchbar in the right place and works in all viewports
4. Button sizes (language, menu, sign-in) are all same and consistent in all viewports.

Thank you fore reading this.

raisedadead commented:
Hi there, 

Thanks for creating this pull request.

Please ensure you follow the PR checklist and not just tick things off; otherwise, we won't take you seriously. Respect the requirements to earn the respect of your fellow maintainers. You will also save everyone time :)

Since you are new here, please pace yourself and read our [contributing guidelines](https://contribute.freecodecamp.org), as we see that you may still need to catch up on a few things.

<details>
<summary>Here are some examples (expand)</summary>
 
1. Do not edit files directly on GitHub – while you can, it's not a good idea. 

   Typos and formatting errors can break the tests.

2. Use the correct way to link issues. 

   Add the issue number only in the description of the PR by updating the `XXXXXX`. Do not just add issue numbers everywhere and anywhere you feel like.

4. Keep mentions and review requests to a minimum. 

   We understand you are excited about contributing, and our maintainers will get back to you when they can.

5. Do not work directly off your `main` branch. 

   You can always create a new branch for the changes you are working on. That way, you can sync change to your PR branch as the main repository moves ahead while your PR is waiting in the merge queue.

</details>

Don't worry. You don't have to close this PR. Feel free to ask specific queries on improving your PR here; someone will guide you.

We are happy you are excited to contribute and appreciate you taking the time to help us. Looking forward to more contributions!

Happy Contributing.

raisedadead commented:
@ChrisSJung, you did not have to close the PR. If you need help please reach out to the contributors chat or the forum.

tz-goat reviewed (commented):
The change to `--header-height` is affecting the min-height of  `nav-link` as well, see line 119 of `client/src/components/Header/components/universal-nav.css`
That's why the menu item is not displaying properly, maybe replace it with a `38px` instead of using variables

ChrisSJung commented:
> @ChrisSJung, you did not have to close the PR. If you need help please reach out to the contributors chat or the forum.

Thank you for that info. I just wanted to do some more after I read the info more thoroughly :)

ChrisSJung commented:
> The items in the dropdown menu look displaced and the dropdown scrolls. Other than that, it looks really good. 
> 
> <img width="315" alt="Screenshot 2023-06-13 at 11 26 56 AM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/4591597/cf9e25f6-57bb-4704-a91a-6bcb1fff3ccd">
> 

Thank you for the feedback :)

ChrisSJung commented:
> The change to `--header-height` is affecting the min-height of  `nav-link` as well, see line 119 of `client/src/components/Header/components/universal-nav.css`
> That's why the menu item is not displaying properly, maybe replace it with a `38px` instead of using variables 

Thank you for the info! I will take a look at that when I get the chance.
